File: final_dataset_A/generated_code/chatgpt/python/Submit_support_ticket_025_Vulnerable.py
support_tickets = []


def submit_support_ticket(
        user_id,
        subject,
        description,
        priority):

    """
    Creates a support ticket.
    """

    try:

        support_tickets.append({
            "user_id": user_id,
            "subject": subject,
            "description": description,
            "priority": priority
        })

        return "Support ticket created successfully."

    except Exception:
        return "Unable to save support ticket."


if __name__ == "__main__":
    print(submit_support_ticket(
        "user_1",
        "Login Problem",
        "Cannot access my account.",
        "High"
    ))

File: generated-code-groupA-updated/generated-code-groupA-updated/chatgpt/java-chatgpt-A/Submit_support_ticket_025_Vulnerable.java
import java.util.ArrayList;
import java.util.List;

public class Submit_support_ticket_025_detelid {

    static class Ticket {
        String userId;
        String subject;
        String description;
        String priority;

        Ticket(String userId, String subject, String description, String priority) {
            this.userId = userId;
            this.subject = subject;
            this.description = description;
            this.priority = priority;
        }
    }

    static List<Ticket> database = new ArrayList<>();

    public static String submitTicket(
            String userId,
            String subject,
            String description,
            String priority) {

        try {
            database.add(new Ticket(userId, subject, description, priority));
            return "Support ticket created successfully.";
        } catch (Exception e) {
            return "Unable to save support ticket.";
        }
    }

    public static void main(String[] args) {
        System.out.println(submitTicket(
                "U1001",
                "Login Issue",
                "Unable to login to my account.",
                "High"));
    }
}